components/dashboard/src/projects/Project.tsx

```diff
@@ -129,7 +129,7 @@ export default function () {
     }
 
     return <>
-        <Header title="Branches" subtitle={<h2 className="tracking-wide">View recent active branches for <a className="text-gray-500 hover:text-gray-800 font-semibold" href={project?.cloneUrl}>{project?.name}</a>.</h2>} />
+        <Header title="Branches" subtitle={<h2 className="tracking-wide">View recent active branches for <a className="gp-link" href={project?.cloneUrl!}>{toRemoteURL(project?.cloneUrl || '')}</a>.</h2>} />
```

Comment:
**issue(non-blocking)**: The blue color seems slightly distracting up there but helpful. Also, this could be resolved once we have a better overview page.